chsdi/views/search.py

```diff
@@ -73,6 +73,23 @@ def search(self):
             self._swiss_search()
         return self.results
 
+    def _fuzzy_search(self, searchTextFinal):
+        # We use different ranking for fuzzy search
+        # For ranking modes, see http://sphinxsearch.com/docs/current.html#weighting
+        self.sphinx.SetRankingMode(sphinxapi.SPH_RANK_SPH04)
+        # Only include results with a certain weight. This might need tweaking
+        self.sphinx.SetFilterRange('@weight', 5000, 2 ** 32 - 1)
```

Comment:
5000 is chosen by hand tests. the other number is simply maximum 32 bit integer (see https://github.com/geoadmin/mf-chsdi3/blob/master/chsdi/lib/sphinxapi/sphinxapi.py#L1134)